data/transit/route/bus.json

```diff
@@ -4708,22 +4692,15 @@
       }
     },
     {
-      "displayName": "Kärnter Linien",
-      "id": "karnterlinien-2dfba8",
-      "locationSet": {"include": ["at"]},
-      "tags": {
-        "network": "Kärnter Linien",
-        "operator": "Kärnter Linien",
-        "route": "bus"
-      }
-    },
-    {
-      "displayName": "Kärntner Linien",
+      "displayName": "Verkehrsverbund Kärnten",
```

Comment:
Hi @kjonosm! As far as I know "Verkehrsverbund Kärnten" is the name of the legal entity behind the network, but as a ground truth there is only the name "Kärntner Linien" to be found. None of the bus tickets or bus stops bear the name "Verkehrsverbund Kärntnen", you'll only see "Kärntner Linien". Even on their webpage, you'll have to stumble upon the legal notice to find "Verkehrsverbund Kärnten".

Reply:
Thanks for the review! As there is no common approach on naming networks in Germany (abbreviated or not?) I decided to close this pull request in lieu of another one with names left untouched but additional wikidata, wikipedia and some minor changes.